src/StatusQ/Components/StatusLoadingIndicator.qml

```diff
@@ -0,0 +1,18 @@
+import QtQuick 2.13
+import StatusQ.Core 0.1
+
+StatusIcon {
+    id: statusIcon
+    icon: "loading"
+    height: 17
+    width: 17
+    RotationAnimator {
```

Comment:
In my opinion, we should take the opportunity that we have here since we are remaking the UI library to change this loading indicator to an animated gif instead of this. We've seen multiple times in the past that having this loading indicator increases CPU usage absurdly, so using a gif instead should fix the problem.

Reply:
That's a very good point. Do we keep `ColorOverlay` capabilities though? We need to stay flexible and be able to apply colors as we see fit

Reply:
Good question, I'm not sure